### PR TITLE
refactor(prom links): simplify renderGrapherImageByChartSlug

### PR DESCRIPTION
**Pull request by mlbrgl**
This should be ok given the single usage of this function so far, which always happens after URLs have been resolved.

### CODE REVIEW

**Comment by danielgavrilov** [comment]
@mlbrgl Is this ready to review, or does it need more?

We could deploy it to a staging site and check the git diff of the static output in case you have any concerns.

**Comment by mlbrgl** [comment]
@danielgavrilov yep should be ok. E2E tests still pass but good to run through the whole site and check for edge cases. Running it now.

**Comment by mlbrgl** [comment]
Adding diff for further review: [diff.txt](https://github.com/owid/owid-grapher/files/8710267/diff.txt)

**Comment by marcelgerber** [comment]
> Adding diff for further review: [diff.txt](https://github.com/owid/owid-grapher/files/8710267/diff.txt)

Analysing the diff, there's only some changed wp container IDs. nothing else has changed.

tip: you can use `git show 4a92b06e2 --word-diff -- *.html` to make this easier for you :)

image

File: baker/siteRenderers.tsx [hunk 1]
```diff
@@ -602,16 +607,18 @@ const getExplorerTitleByUrl = async (url: Url): Promise<string | undefined> => {
...
 }
 
-const renderGrapherImageByChartSlug = async (
+/**
+ * Renders a chart thumbnail given a slug. The slug is considered "resolved",
+ * meaning it has gone through the internal URL resolver and is final from a
+ * redirects perspective.
+ *
+ * @param chartSlug
+ * @returns
```
Comment (danielgavrilov) [inline baker/siteRenderers.tsx:616]: nit: param/returns are already expressed in TypeScript? I think probably VSCode automatically added it to "help you out"

**Review by danielgavrilov — APPROVED** [review]

**Comment by mlbrgl** [comment]
@marcelgerber thanks for the tip!

**Comment by mlbrgl** [comment]
`git diff` on live is as expected: https://github.com/owid/owid-static/commit/c88d9886366c0ca4566e1207907377db102e1d68
